user-interface/src/main/java/life/qbic/datamanager/views/general/spreadsheet/ColumnValidator.java

```diff
@@ -0,0 +1,47 @@
+package life.qbic.datamanager.views.general.spreadsheet;
+
+import java.util.function.Predicate;
+
+/**
+ * Validates the content of a cell. Can be applied to a column.
+ * <p>
+ * One column can have multiple validators. The spreadsheet can use the validators to evaluate the
+ * validity of cells in the column.
+ */
+public class ColumnValidator<T2> {
```

Comment:
NIT: why T2 and not just T?

Reply:
Can be `T`. :+1: This class was originally in the Spreadsheet class. The spreadsheet class provides a typed parameter `T`. Originally `T2` was chosen to avoid conflicts with `T`. As the class `ColumnValidator` moved out of the `Spreadsheet` class, `T` can now be used.